components/dialog/overview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Overview
+page_title: Dialog Overview
+description: Overview of the Dialog for Blazor.
+slug: dialog-overview
+tags: telerik,blazor,dialog,overview
+published: True
+position: 0
+---
+
+# Dialog Overview
+
+The Dialog component is a modal popup that displays information to the user. It is fully customizable and extends the `DialogBuilder` to provide you with a way to achieve complex UI popups.
+
+## Basics
+
+To add a Telerik Dialog to your Blazor app:
+
+1. Add the `TelerikDialog` tag.
+1. Provide `Visible` parameter.
+1. Set its `Title`.
+1. Optionally, choose a `ButtonsLayout` (from the `DialogButtonsLayout` enums).
+    * Its default layout is `DialogButtonsLayout.Stretched`. See more in the [Action Buttons article]({%slug  dialog-action-buttons%})).
+1. Optionally, set its `ShowCloseButton` parameter (true/false).
+    * Its **true** by default. See more in the [Header article]({%slug  dialog-header%}).
+
+The following example demonstrates how to set up the Dialog with its default configuration.
+
+>caption A basic configuration of the Telerik Dialog.
+
+````CSHTML
+@* An example of the Dialog basic implementation. *@
+
+<TelerikDialog @bind-Visible="@Visible"
+               Title="@Title">
+    <DialogContent>
+        <br />
+        <div>--- Place here the Dialog content ---</div>
+        <br />
+    </DialogContent>
+    <DialogButtons>
+        <TelerikButton Primary="true" OnClick="@(() => { Visible = false; })">Cancel</TelerikButton>
+        <TelerikButton OnClick="@(() => { Visible = false; })">OK</TelerikButton>
```

Comment:
A primary "cancel" button is suitable only for destructive operations and may look strange here. Consider using a standard example with primary "OK".